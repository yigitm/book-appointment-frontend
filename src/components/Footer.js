import React from 'react';

const Footer = () => (
  <div className="bg-gray-400 h-12 md:hidden">Mobile Footer</div>
);

export default Footer;
