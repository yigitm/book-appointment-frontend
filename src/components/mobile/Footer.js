import React from 'react';
import {
  FaSignature,
  FaTwitter,
  FaFacebookF,
  FaGooglePlusG,
  FaVimeoV,
  FaPinterestP,
} from 'react-icons/fa';

const Footer = () => (
  <div className="flex justify-center items-center">
    <FaTwitter className="mr-1" />
    <FaFacebookF className="mr-1" />
    <FaGooglePlusG className="mr-1" />
    <FaVimeoV className="mr-1" />
    <FaPinterestP className="mr-1" />

    <p className="text-sm">@2022 microverse-inc</p>
  </div>
);

export default Footer;
