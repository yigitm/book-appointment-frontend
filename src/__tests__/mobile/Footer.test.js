import React from 'react';
import Footer from '../../components/mobile/Footer';

describe('Mobile Footer render', () => {
  test('Mobile Footer', () => {
    const Footer = () => <Footer />;

    expect(Footer()).toMatchSnapshot();
  });
});
