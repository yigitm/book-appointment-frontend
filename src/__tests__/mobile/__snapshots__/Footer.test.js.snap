// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mobile Footer render Mobile Footer 1`] = `<Footer />`;
